### ISSUE TITLE
MIRnet nan problem

### ISSUE THREAD

**Issue by soheilmajidi** [keras-team/keras-io#750]
hello ,
I got a problem that I couldn't solve .
I wanted to use your model for image deblurring . my train data is 64*64*3 blurry data and my test data is clear 64*64*3.
After 12 or 13 epoch I get nan in Charbonnier loss and PSNR . I have tried your model in denoising task and it worked great . I changed last layers' activation function from softmax to sigmoid but again I got the nan . 
would you please recommend s.th to make my model run accurately please ?

**Comment by sachinprasadhs** [comment]
Hi,

For better understanding, could you please provide reproducible code. 

Refer to the latest **Denoising Diffusion Implicit Models** tutorial here https://keras.io/examples/generative/ddim/
